src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateCreationTests.Responses.cs

```diff
@@ -276,7 +277,7 @@ async ValueTask<Todo> ValueTaskTestActionAwaited()
         }
     }
 
-    [Theory]
+    [ConditionalTheory(Skip = "https://github.com/dotnet/runtime/issues/87073")]
```

Comment:
Your change is regressing this scenario because we are now ALWAYS calling `GetTypeInfo(typeof(object))`? Is that correct? (this test is passing in main) Maybe we shouldn't be doing this change if it is going to regress scenarios. I just thought you were hitting issues in new tests you were converting (for filters).

Reply:
> Your change is regressing this scenario because we are now ALWAYS calling GetTypeInfo(typeof(object))? Is that correct?

Yes, although it's only regressing the scenario when a JsonContext is used.

> Maybe we shouldn't be doing this change if it is going to regress scenarios.

I think we should still do it, but I'm comfortable waiting until the bug fix in the runtime is merged to merge this in to our repo.

> I just thought you were hitting issues in new tests you were converting (for filters).

The filters scenario was to showcase that the bug was happening in RDF as well and wasn't a side-effect of RDG's codegen, but as we eventually realized, any invocation of `GetTypeInfo(typeof(object))` will trigger this at the moment.